wrap-battle/src/hooks/useRoom.ts

```diff
@@ -3,26 +3,39 @@ import { useParams } from 'react-router-dom';
 
 import { firestore as db } from '../services/firestore';
 import { Room } from '../types/room';
+import { Player } from '../types/player';
 
-interface roomParams {
+type roomParams = {
     id: string;
-}
+};
 
-interface Output {
+type roomOutput = {
     room?: Room;
-}
+    roomId?: string;
```

Comment:
@hierischmarkus Ich hab in dem useRoom hook noch ein paar weitere output-variablen hinzugefügt. Dadurch brauchst du die Logik nicht auch implementieren, sondern kannst einfach die einzelnen Variablen aus dem hook verwenden. Sollte uns zumindest teilweise ein paar ? ersparen 😛 

Beispiel: `const { players, playerOnTurn } = useRoom();`

Reply:
Taugt mir